docs/getting-started/network-page-overview.asciidoc

```diff
@@ -39,7 +39,7 @@ to configure IP reputation links.
 [discrete]
 [[map-ui]]
 == Map
-The map provides a visual overview of your network traffic. It is interactive, so you can start exploring data directly from the map. Hover over source and destination points to see more information, such as hostnames and IP addresses.
+The map provides a visual overview of your network traffic. It is interactive, so you can start exploring data directly from the map. Hover over source and destination points to view more information, such as hostnames and IP addresses.
```

Comment:
```suggestion
The map provides a visual overview of your network traffic. It is interactive, so you can start exploring data directly from the map. Hover over source and destination points to observe more information, such as hostnames and IP addresses.
```

Reply:
I've seen @joepeeples use this a few times and think it's a great alternative.

Reply:
Have I? Lol, I don't think I've been consciously doing it, but it sounds like a good alternative. I don't have a big problem with "view" either though, especially since the page is a "visual overview."

Reply:
Yeah, I have no objections to view/overview in the first sentence since it's being used as noun. I've been trying to find alternatives to "view" when it's used as a verb though. I actually like "observe" because I think it encompasses the other senses that you can use to perceive something. For example, you can use a screen reader to identify certain UI elements and observe (hear) table data in some Security app pages.